ml-algorithms/src/main/java/org/opensearch/ml/engine/ModelHelper.java

```diff
@@ -86,13 +86,13 @@ private void verifyModelZipFile(String modelZipFilePath) throws IOException {
                 String fileName = ((ZipEntry) zipEntries.nextElement()).getName();
                 if (fileName.endsWith(PYTORCH_FILE_EXTENSION) || fileName.endsWith(ONNX_FILE_EXTENSION)) {
                     if (hasModelFile) {
-                        throw new IllegalArgumentException("Find multiple model files");
+                        throw new IllegalArgumentException("Find multiple model files, but expected only one");
                     }
                     hasModelFile = true;
                 }
                 if (fileName.equals(TOKENIZER_FILE_NAME)) {
```

Comment:
how can we have two files with the same name `tokenizer.json`?

Reply:
Good point, planned to check nested subfolder, should be `endsWith`. Now the code doesn't support nested subfolders in model zip. So we can remove this check now. Will fix this in next PR.